indico/modules/categories/templates/management/settings.html

```diff
@@ -0,0 +1,66 @@
+{% from 'forms/_form.html' import form_header, form_rows, form_fieldset, form_footer %}
+
+{% extends 'categories/management/base.html' %}
```

Comment:
extends should be the first line/statement (even though having imports first doesn't cause any issues)